pkg/services/libraryelements/model/model.go

```diff
@@ -20,8 +20,9 @@ const (
 
 // LibraryElement is the model for library element definitions.
 type LibraryElement struct {
-	ID          int64  `xorm:"pk autoincr 'id'"`
-	OrgID       int64  `xorm:"org_id"`
+	ID    int64 `xorm:"pk autoincr 'id'"`
+	OrgID int64 `xorm:"org_id"`
+	// Deprecated: use FolderUID instead
```

Comment:
does it also have to get added to the swagger bits?

Reply:
This is one of a series of PRs to help removing folderID. This specific struct is a database model and it's not related to swagger (yet). In swagger definitions we have both folderUID and folderID for a long time (not sure about deprecation), but in API docs on the website we have folderID deprecated since v9:

> folderId: ID of the folder where the library element is stored. It is deprecated since Grafana v9

Reply:
hi ryan, sorry i don't quite follow - what do you mean swagger wise?

Reply:
I think we need to introduce the `FolderUID` field too (it does not exist right now)

Reply:
Not really, we need the migration first otherwise the column would be missing on inserts. FolderUID field would be part of my following PR instead.